boards/arm/nucleo_g071rb/doc/index.rst

```diff
@@ -100,6 +100,8 @@ The Zephyr nucleo_g071rb board configuration supports the following hardware fea
 +-----------+------------+-------------------------------------+
 | CLOCK     | on-chip    | reset and clock control             |
 +-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | rtc                                 |
```

Comment:
Please remove the conflicts lines in the commit message:
```
boards: arm: stm32g071 add rtc feature on the nucleo board

This patch enables the rtc so that the testcase
tests/drivers/counter/counter_basic_api
can run on this nucleo_g071rb board
also when running sanity check

Signed-off-by: Francois Ramu <francois.ramu@st.com>

# Conflicts:
#	boards/arm/nucleo_g071rb/nucleo_g071rb.yaml
```